bundles/binding/org.openhab.binding.zwave/database/aeon/zw056.xml

```diff
@@ -1,119 +1,123 @@
-                        <Item>
-                                <Value>0</Value>
-                                <Label lang="en">Ignore</Label>
-                        </Item>
-                        <Item>
-                                <Value>1</Value>
-                                <Label lang="en">Play</Label>
-                        </Item>
-                        <Item>
-                                <Value>2</Value>
-                                <Label lang="en">Stop</Label>
-                        </Item>
-                        <Item>
-                                <Value>3</Value>
-                                <Label lang="en">Pause</Label>
-                        </Item>
-                        <Item>
-                                <Value>4</Value>
-                                <Label lang="en">Next</Label>
-                        </Item>
-                        <Item>
-                                <Value>5</Value>
-                                <Label lang="en">Previous</Label>
-                        </Item>
-                        <Label lang="en">Send Basic Set on open/close event</Label>
-                        <Help lang="en">Default=0</Help>
-                </Parameter>
-                <Parameter>
-                        <Index>8</Index>
-                        <Type>long</Type>
-                        <Default>10</Default>
-                        <Size>1</Size>
-                        <Minimum>0</Minimum>
-                        <Maximum>10</Maximum>
-                        <Label lang="en">Ringtone volume</Label>
-                        <Help lang="en">Default=10</Help>
-                </Parameter>
-                <Parameter>
-                    <Index>80</Index>
-                    <Type>byte</Type>
-                    <Default>25</Default>
-                    <Minimum>0</Minimum>
-                    <Maximum>60000</Maximum>
-                    <Size>1</Size>
-                    <Label lang="en">Enable Notifications</Label>
-                    <Help lang="en">Enable to send notifications to associated devices (Group1) when the state of Siren changed (0=nothing,1=hailCC,2=basic CC report).</Help>
-                </Parameter>
-    </Configuration>
-    
-    <Associations>
-        <Group>
-            <Index>1</Index>
-            <Maximum>5</Maximum>
-            <Label lang="en">Group 1</Label>
-            <SetToController>true</SetToController>
-        </Group>
-    </Associations>
+	<Model>ZW056-A</Model>
+	<Label lang="en">Doorbell</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x25</id></Class>
+		<Class><id>0x59</id></Class>
+		<Class><id>0x5E</id></Class>
+		<Class><id>0x70</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x73</id></Class>
+		<Class><id>0x82</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Associations>
+		<Group>
+			<Index>1</Index>
```

Comment:
I'm not sure, the manual is kinda useless when trying to determine the command classes. I found a document that showed the hex codes with the english class names. But as far as what reports to the controller i don't see anything that says that anywhere, or I don't know what I am looking for if something does say that. Guessing may 0x82 as that is hail CC, but thats simply a guess.